npm/webpack-dev-server/test/handlers/angularHandler.spec.ts

```diff
@@ -38,6 +38,7 @@ describe('angularHandler', function () {
     expect(webpackConfig).to.exist
     expect((webpackConfig?.entry as any).main).to.be.undefined
     expect(sourceWebpackModulesResult.framework?.importPath).to.include(path.join('@angular-devkit', 'build-angular'))
+    expect(webpackConfig.stats).to.equal('errors-only')
```

Comment:
@AtofStryker Should there be a test ensuring that if it's overridden with verbose that it's read in as verbose?

Reply:
I should also mention that the devkit defaults could change at any time and the test could break if we test the inverse, though I suspect that is highly unlikely